src/subscriber/event/UpdateEvent.ts

```diff
@@ -37,16 +37,13 @@ export interface UpdateEvent<Entity> {
     databaseEntity: Entity;
 
     /**
-     * List of updated columns.
+     * List of updated columns. In query builder has no affected
      */
     updatedColumns: ColumnMetadata[];
 
     /**
-     * List of updated relations.
+     * List of updated relations. In query builder has no affected
```

Comment:
because it's doesn't work here
https://github.com/typeorm/typeorm/blob/80e82f4a42299656adb68fb119a6fa1c10156e38/src/query-builder/UpdateQueryBuilder.ts#L67

Reply:
@pleerock what can you say?